setup.cfg

```diff
@@ -29,8 +29,8 @@ packages = find:
 python_requires = >=3.7
 install_requires =
 	qiskit >= 0.34.2
-	qiskit-terra>=0.19.2
-	pyqir-generator >= 0.4.0a1
+	qiskit-terra >= 0.19.2
+	pyqir-generator == 0.4.2a1
```

Comment:
should this be `>=` , in case we want to publish a new pyqir version without having to make cascading updates?

Reply:
@anpaz I think we should still lock this version down.